model_zoo/lbann.cpp

```diff
@@ -132,10 +132,13 @@ int main(int argc, char *argv[]) {
     }
     El::ReportException(e);
     return EXIT_FAILURE;
-  } catch (std::exception& e) {
+  }
+#if 0
```

Comment:
Is this to allow normal exceptions to propagate all of the way out?

Reply:
That was an accidental commit, but yes. I had some `bad_cast` problems I was debugging. I'll remove this.